handbook/engineering/distribution/index.md

```diff
@@ -2,101 +2,54 @@
-- **Observability: Monitoring** ("site admins should easily know the health of Sourcegraph")
-    - Monitoring & alerting infrastructure
-    - Educating site admins about how to monitor Sourcegraph
-    - Working with & ensuring engineering @ Sourcegraph adds needed monitoring
-    - **Primary owners**: @stephen, @uwe
-    - **Related code**: [monitoring generator (dashboards/alerts)](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/tree/monitoring), [Grafana docker image](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/tree/docker-images/grafana), [Prometheus docker image](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/tree/docker-images/prometheus)
-- **Observability: Debugging** ("site admins should be able to collect the information needed to debug issues easily")
-    - Logging & Tracing infrastructure
-    - Working with & ensuring engineering @ Sourcegraph adds needed logging/tracing to debug issues
-    - Making the debugging process for common problems seamless and straightforward
-    - Making reporting issues with all needed information easy
-    - Ensuring logs/tracing are not overly verbose, identify most useful information for solving problems
-    - **Primary owners**: @stephen
-    - **Related code**: [Jaeger Docker images and code](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+file:jaeger&patternType=literal), [opentracing code (broadly)](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+opentracing&patternType=literal), Jaeger [k8s](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/deploy-sourcegraph%24+jaeger&patternType=literal), [docker-compose/pure-docker](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/deploy-sourcegraph-docker%24+jaeger&patternType=literal), and [single-container](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+file:cmd/server+jaeger&patternType=literal) deployments & [associated docs](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+file:doc/admin/observability+jaeger%7Ctracing&patternType=regexp)
-
-## Tech stack
-
-Go, Docker, Kubernetes
-
-## Details
-
-* [Recurring processes](./recurring_processes.md)
-* [Internal infrastructure](./internal_infrastructure.md)
-* [Tools](./tools/index.md)
-* Tutorials
-  * [Observability developer guide](observability/index.md)
-  * [How to set up a separate website maintained by Sourcegraph](separate_website.md)
-  * [How to replay a metrics dump from a customer](use_metrics_dump.md)
-  * [How to simulate k8s admin security restrictions](k8s_admin_custom_policy.md)
-  * [How to test the Gitlab native integration locally](gitlab_native_local.md)
-  * [How to make updates to global settings and configuration on sourcegraph.com](update_sourcegraph_website.md)
+## Where to find us
```

Comment:
I would like to keep team headings consistent across team pages. If you want to edit it here then please update the other team pages too.

Reply:
Cool, I will try to do that. FWIW, it appears this it is not consistent at all today either even without my change:

- Code intel: Vision, Direction, Contact, Tech Stack, ...
- Cloud: Contact, Resources, Processes, Members, ...
- Campaigns: Vision, Mission, Analgoies, Members, ...
- Search: Vision, Contact, Project Leads, Members, ...
- Security: Vision, Contact, How to report a security vulnerability, ...

Reply:
I don't mean that *every* heading has to be the same, but for the common ones (like team and contact) I would like to keep them the same. Yes, there is some debt here but I don't want that debt to increase :)

Reply:
Great, makes sense. Just to be clear, the heading here was "Team API" before my change which I thought was super weird, so I reworded it. I agree consistency here is important though, will update 👍

Reply:
```suggestion
## How to contact us
```